pkg/smartcontract/manifest/manifest.go

```diff
@@ -32,6 +32,8 @@ type Manifest struct {
 	Name string `json:"name"`
 	// ABI is a contract's ABI.
 	ABI ABI `json:"abi"`
+	// Features is a set of contract features. Currently unused.
+	Features map[string]interface{} `json:"features"`
```

Comment:
Could you, please, store it as `json.RawMessage` to preserve the order of JSON fields? It's the same problem as the one mentioned in https://github.com/neo-project/neo/issues/2181 and https://github.com/neo-project/neo/issues/2345.